README.md

```diff
@@ -1 +1,40 @@
-# oscal-sdk-go
\ No newline at end of file
+# oscal-sdk-go
+
+`oscal-sdk-go` complements the `compliance-trestle` SDK by providing the core SDK functionality in Go.
+
+> WARNING: This project is currently under initial development. APIs may be changed incompatibly from one commit to another.
+
+## Supported Functionality
+
+Below is a table to show what is currently supported by the SDK.
+
+| SDK Functionality                         | Supported |
+|-------------------------------------------|-----------|
+| OSCAL Types with Basic Trestle Extensions | &#10003;  |
+| OSCAL Schema Validation                   |           |
+| Target Components Extension               |           |
+| Multiple Parameters per Rule              |           |
+| OSCAL to OSCAL Transformation             |           |
+| OSCAL Constraints Validation              |           |
+
+## Run tests
+
+```bash
+make test-unit
+```
+
+## Format and Style
+
+**Requires [`golangci-lint`](https://golangci-lint.run/welcome/quick-start/)**
+
+```bash
+make fmt
```

Comment:
Should this be `make format`?

This is a just minor comment so no need to address now.